platforms/cpuinfo.go

```diff
@@ -129,3 +140,16 @@ func getCPUVariant() string {
 
 	return variant
 }
+
+// getCPUArchitecture from uname -m get CPU Architecture
+func getCPUArchitecture() (string, error) {
+	if !isLinuxOS(runtime.GOOS) {
+		return "", fmt.Errorf("getCPUArchitecture for OS %s: %w", runtime.GOOS, errdefs.ErrNotImplemented)
+	}
+	var name unix.Utsname
+	err := unix.Uname(&name)
+	if err != nil {
+		return "", err
+	}
+	return string(name.Machine[:bytes.IndexByte(name.Machine[:], 0)]), nil
```

Comment:
This returns a value like “armv7l” while the OCI arch variant must be like “v7”

Reply:
The `getCPUVariant` function only returns a value similar to `armv7l`, and the arm arch variant is processed by the `getCPUVariant` function.

Reply:
Where is “armv7l” parsed?